task/python-boto3/0.1/python-boto3.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         name: python-script-configmap
   steps:
     - name: run-python-script
-      image: python:3.9
+      image: docker.io/python:3.9
```

Comment:
`docker.io/library/python:3.9` would work 🙃 , but I don't think this one does.

Reply:
@vdemeester I am using this 
python:3.9.19-alpine3.20@sha256:45cc18540209d878c2b24080cf8f64fc37603721b67d0ecc508799e2f9a9b21d

lighter image with sha